gbench/clblast/cli_device_selector.hpp

```diff
@@ -0,0 +1,107 @@
+/* Copyright (c) 2015 The Khronos Group Inc.
```

Comment:
@mehdi-goli @Ruyk Is this license correct? I believe this file has been copied/moved/adapted from the old benchmarks, but I don't know if the copyright is strictly correct, or if it should be copyright Codeplay.